runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -491,6 +491,18 @@ public String getUrl() {
     }
 
     /**
+     * Get the content height of current web page/app.
+     * NOTE: When page finished loading, it's content size info may not be updated.
```

Comment:
So does this mean when finished loading, its OK to call this? Otherwise may get a invalid data?